src/overmap.h

```diff
@@ -48,6 +48,7 @@ struct city {
     }
 
     int get_distance_from( const tripoint &p ) const;
+    tripoint position() const;
```

Comment:
Even better, if you use overmapbuffer::closest_city(), the reference it returns has this position already.

Reply:
Will try to do that, though `position` in `city_reference` is in submap coordinates, not omt coordinates.